test/managers/course_repository_test.dart

```diff
@@ -1,48 +1,38 @@
 // FLUTTER / DART / THIRD-PARTIES
 import 'dart:convert';
-
 import 'package:flutter_test/flutter_test.dart';
 import 'package:intl/intl.dart';
 import 'package:mockito/mockito.dart';
 
 // SERVICES / MANAGER
 import 'package:notredame/core/services/analytics_service.dart';
-import 'package:notredame/core/services/signets_api.dart';
 import 'package:notredame/core/managers/user_repository.dart';
 import 'package:notredame/core/managers/cache_manager.dart';
 import 'package:notredame/core/managers/course_repository.dart';
+import 'package:ets_api_clients/clients.dart';
 
 // MODELS
-import 'package:notredame/core/models/schedule_activity.dart';
-import 'package:notredame/core/models/session.dart';
-import 'package:notredame/core/models/course_activity.dart';
-import 'package:notredame/core/models/mon_ets_user.dart';
-import 'package:notredame/core/models/course.dart';
-import 'package:notredame/core/models/course_summary.dart';
-import 'package:notredame/core/models/evaluation.dart' as model;
-import 'package:notredame/core/models/course_evaluation.dart';
-
-// CONSTANTS
-import 'package:notredame/core/constants/activity_code.dart';
+import 'package:ets_api_clients/models.dart';
+import 'package:ets_api_clients/models.dart' as models;
```

Comment:
Is this necessary? I see `models` being used at lines 1260 and 1828, would it be possible to just write `Evaluation(` instead of `models.Evaluation(` and remove the duplicated import statement?